lib/rrule/rule.rb

```diff
@@ -84,7 +69,7 @@ def all_until(end_date: nil, limit: nil)
       end
 
       loop do
```

Comment:
Should we be rejecting exdates inside this loop? Otherwise it looks like enumerating will include exdates? Perhaps a bit of a product question, but I wouldn't expect exdates to be yielded.

Reply:
Oooh good point. Right now exdates are only being rejected in `all` and `between` - the enumerable interface should definitely exclude these also.

Reply:
Fixed - as a side benefit now we don't have to use `reject_exdates` which always struck me as kinda ugly. Thanks for the catch!